File: src/question/day0629/ms_0629.cs
using System;

namespace algorithm_01
{
    class Program
    {
        static void Main(string[] args)
        {
            var kor = 90;
            var mat = 80;
            var eng = 100;
            var sum = kor + mat + eng;
            Console.WriteLine($"kor {kor}\nmat {mat}\neng {eng}\nsum {sum}\navg {sum/3}");
        }
    }
}

File: year2020/month06/day0629/ms_0629.cpp
#include <iostream>

using namespace std;

int main() {
	int kor = 90, mat = 80, eng = 100;
	int sum = kor + mat + eng;
	int avg = sum / 3;

	cout << "kor " << kor << "\nmat " << mat << "\neng " << eng << "\nsum " << sum << "\navg " << avg << endl;
	return 0;
}